dbt/models/shared_columns.md

```diff
@@ -1300,9 +1301,27 @@ with `sv_is_ptax_outlier` (using OR logic).
 NOTE: Outlier flags only exist for sales _after_ 2014.
 {% enddocs %}
 
-## sv_outlier_type
+## sv_outlier_reason1
+
+{% docs shared_column_sv_outlier_reason1 %}
+Heuristic or model used to flag an outlier.
+
+See the [model-sales-val](https://github.com/ccao-data/model-sales-val) repo
+for a list of possible flags.
+{% enddocs %}
```

Comment:
**[Nitpick, non-blocking]** Same here, let's consolidate these into one `shared_column_sv_outlier_reason` (i.e. just rename `shared_column_sv_outlier_type` to `shared_column_sv_outlier_reason`).